strawberryfields/apps/graph/utils.py

```diff
@@ -227,8 +227,8 @@ def c_1(clique: list, graph: nx.Graph):
         graph (nx.Graph): the input graph
 
     Returns:
-       list[tuple(int)]: A list of tuples. The first node in the tuple is the node in the clique and the
-       second node is the outside node it can be swapped with.
+       list[tuple(int)]: A list of tuples. The first node in the tuple is the node in the clique
+       and the second node is the outside node it can be swapped with.
```

Comment:
what was the reason for this change?

Reply:
🤔 
It was actually longer than 100 characters but had somehow slipped through the net.
IIRC that line had been changed from a github suggestion, where it is sometimes hard to see the max line width